synapse/storage/databases/main/event_push_actions.py

```diff
@@ -758,7 +803,10 @@ def get_no_receipt(
                 actions=_deserialize_action(row[3], row[4]),
                 received_ts=row[5],
             )
-            for row in after_read_receipt + no_read_receipt
+            for row in push_actions
```

Comment:
ditto for destructuring here

Reply:
https://github.com/matrix-org/synapse/pull/13597/commits/b7a66e330afa97d686a0f38e12caaf1308f583d6